lib/jobs/base-job.js

```diff
@@ -569,7 +569,7 @@ function baseJobFactory(
      */
     BaseJob.prototype._publishRunRedfishCommand =
         function _publishRunRedfishCommand (uuid, hostData) {
-        return messenger.publish(
+        return messenger.publishInternalEvents(
             Constants.Protocol.Exchanges.Task.Name,
```

Comment:
Those two messenger.publish should be move to on-core/lib/protocol/events, otherwise you need another change here to align data format. 
For this PR I am OK to keep as it is